coderd/audit/request.go

```diff
@@ -238,3 +248,30 @@ func parseIP(ipStr string) pqtype.Inet {
 		Valid: ip != nil,
 	}
 }
+
+type GroupMemberLists struct {
+	OldGroupMembers []string
+	NewGroupMembers []string
+}
+
+// Adds a 'members' key to Group resource diffs
+// in order to capture the addition or removal of group members
+func addGroupMemberDiff(logCtx context.Context, diff Map, groupMemberLists json.RawMessage, logger slog.Logger) Map {
+	var (
+		groupMemberBytes = []byte(groupMemberLists)
+		members          GroupMemberLists
+		err              = json.Unmarshal(groupMemberBytes, &members)
+	)
+
+	if err == nil {
+		diff["members"] = OldNew{
+			Old:    members.OldGroupMembers,
+			New:    members.NewGroupMembers,
+			Secret: false,
+		}
+	} else {
+		logger.Warn(logCtx, "marshal group member diff", slog.Error(err))
+	}
```

Comment:
Is there a better way to log here so that I don't have to pass through `logCtx` and `logger`?

Reply:
Nah, it's on purpose though. The context can potentially hold fields that the logger can use.